relay.go

```diff
@@ -467,12 +474,16 @@ func (r *Relayer) handleCallReq(f *lazyCallReq) error {
 
 	f.Header.ID = destinationID
 	call.SentBytes(f.Frame.Header.FrameSize())
```

Comment:
we're no longer tracking sent bytes accurately when appends are involved.

can you check if this is easy to fix as part of this change, and if not, add a TODO and fix as a follow up?